src/cljs/rems/spa.cljs

```diff
@@ -411,13 +399,32 @@
 ;; must be called after routes have been defined
 
 (defn hook-browser-navigation! []
-  (doto (History.)
-    (events/listen
-     HistoryEventType/NAVIGATE
-     (fn [event]
-       (js/window.rems.hooks.navigate (.-token event))
-       (secretary/dispatch! (.-token event))))
-    (.setEnabled true)))
+  (events/listen accountant/history
+                 HistoryEventType/NAVIGATE
+                 (fn [event]
+                   (js/window.rems.hooks.navigate (.-token event))))
```

Comment:
why not combine these two `event/listen` calls? or at least put them next to each other?

Reply:
The first is called when the page is initially loaded, the second is not. I'll add one more comment.